.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Submit a bug request
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+*Please do not include your AWS account or credentials as part of the bug request.*
```

Comment:
It may be better to use the invisible rendering hack rather than include advise and placeholders in the actual rendered template. That way users don't have to delete the lines (or if they don't it makes the issue harder to parse)

```
[//]: # (Please do not include your AWS account or credentials as part of the bug request.)
```